### ISSUE TITLE
How is the performance on Pascal VOC？

### ISSUE THREAD

**Issue by StillWaterRUN** [hustvl/YOLOS#4]
## ❔How is the performance on Pascal VOC？
Hi, I test YOLOS on pascal voc 2007 with default parameters, I can't get a satisfactory result, here is my result：

 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.276
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.497
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.274
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.006
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.085
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.390
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.297
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.433
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.490
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.053
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.289
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.628

I wonder is there anything go wrong, could you give me some advice?

**Comment by LegendBC** [comment]
Hi @StillWaterRUN , thanks for your interest on YOLOS, we have not run experiments on VOC2017, so can you provide us with more details in your training?(which model, how many epochs you run, from scratch or finetune

**Comment by StillWaterRUN** [comment]
Hi @LegendBC , I finetuned YOLOS-B on VOC2007, the backbone is pretrained on ImageNet1K (DeiT-base distilled 384), total epoch: 130, here is my training config:
backbone_name='base', batch_size=2, bbox_loss_coef=5, clip_max_norm=0.1, coco_panoptic_path=None, coco_path='./Dataset/voc0712_coco', dataset_file='voc', decay_rate=0.1, det_token_num=100, device='cuda', dice_loss_coef=1, dist_backend='nccl', dist_url='env://', distributed=True, eos_coef=0.1, epochs=150, eval=False, eval_size=800, giou_loss_coef=2, gpu=0, init_pe_size=[800, 1344], lr=2.5e-05, lr_backbone=1e-05, lr_drop=100, mid_pe_size=[800, 1344], min_lr=1e-07, num_workers=2, output_dir='./output/box_model', pre_trained='/home/LiaoMingxiang/Workspace/pretrained/deit_base_distilled_patch16_384-d0272ac0.pth', rank=0, remove_difficult=False, resume='', sched='warmupcos', seed=42, set_cost_bbox=5, set_cost_class=1, set_cost_giou=2, start_epoch=0, use_checkpoint=False, warmup_epochs=0, warmup_lr=1e-06, weight_decay=0.0001, world_size=4

**Comment by StillWaterRUN** [comment]
I think the small amount of data of VOC2007 should be responsible for the performance.

**Comment by Yuxin-CV** [comment]
We believe we have answered your question, and as such I'm closing this issue, but let us know if you have further questions.

**Comment by hyunOO** [comment]
Hello @StillWaterRUN , could you share the configurations of the experiments (e.g, the architecture you used, types of augmentations, # of detection tokens ... ) you conducted on VOC 2007?
